### DISCUSSION TITLE
How to solve the amplitude integral?

### DISCUSSION THREAD

**Discussion by marcelogcampos** [FeynCalc/feyncalc#228] [Q&A] [answered]
Hi FeynCalc developers,

We'd (from Brazil) like to thank you for the open source package. 
We are beginners with FeynCalc and we are using it to calculate a glueball process involving two ingoing particles (quark and antiquark "c", with four momenta p1 and p2) and four outgoing (quark and antiquark "u" and "d", with four momenta k1, k2, k3 and k4), with one loop (four momentum q) compound by two quark-gluon three-vertices (c-c-g) and two gluon-gluon three-vertices, and, besides, there are two extra quark-gluon three-vertices (u-u-g and d-d-g) after the loop. We executed the FCFAConvert instruction and had the amplitude integral below:

~ⅈ gs^2 g^{Lor1 Lor2} g^{Lor10 Lor9} g^{Lor3 Lor4} g^{Lor5 Lor6} g^{Lor7 Lor8}
f^{ Glu7 Glu9 Glu11} f^{Glu8 Glu10 Glu11}  T_{Col5 Col6}^{Glu8}  T_{Col3 Col4}^{Glu7}  T_{Col2 Col7}^{Glu10}  T_{Col7 Col1}^{Glu9}
(g_{Lor4 Lor8} (k3 + k4 + p2 + q)_{Lor10} + 
g_{Lor10 Lor8} (k3 + k4 - 2p2 - 2q)_{Lor4} + 
g_{Lor10 Lor4} (-2k3 - 2k4 + p2 + q)_Lor8) 
(g_{Lor6 Lor9} (-k1 - k2 - 2k3 - 2k4 + 2p2 + 2q)_{Lor2} + 
g_{Lor2 Lor9} (-k1 - k2 + k3 + k4 - p2 - q)_{Lor6} + 
g_{Lor2 Lor6} (2k1 + 2k2 + k3 + k4 - p2 - q)_{Lor9})
(φ ( k3 , md )) . (-ⅈ gs γ_{Lor3}) . (φ ( -k4 , md ))
(φ ( k1 , mu )) . (-ⅈ gs γ_{Lor1}) . (φ ( -k2 , mu ))
(φ ( -p2 , mc )) . (-ⅈ gs γ_{Lor7} ) . (γ ·q + mc) . 
(-ⅈ gs γ_{Lor5}) . (φ ( p1 , mc ))) / 
(16 π^4 ( -k1 - k2 )^2  (-k3 - k4 )^2  (q^2 - mc^2) . (p2 + q )^2 . (-k3 - k4 + p2 + q )^2 . (-k1 - k2 - k3 - k4 + p2 + q )^2)))

Now we are figuring out how to solve this expression through FeynCalc. We've seen some examples at the gallery and observed the utilization of Mandelstam variables and the Passarino-Veltman functions. Is it mandatory to follow these two steps in FeynCalc?  Could you help us suggesting some typesettings and directions to solve this amplitude? 
Our FeynCalc version is 9.3.1, FeynArts 3.11 and Mathematica 10.

Best regards,
marcelo.

**Comment by vsht** [comment, marked as answer]
Hi,

you've already mentioned that you found the examples. In addition to that, there is also
a manual containing a tutorial:

https://github.com/FeynCalc/feyncalc-manual/releases/download/dev-manual/FeynCalcManual.pdf

This should be more than enough information for you to learn how to use FeynCalc to get your
calculation done.

Cheers,
Vlad

**Reply by marcelogcampos** [reply]
Hi Vlad,

thanks for your help and time! 

marcelo.
